### PR TITLE
multiple cars, same name, add vin next to name

### LINKED ISSUE #3891: Same name cars issue, can you  use ID-CarName or CarName (ID) on the dropdown, it's only showing 1 car
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

2 cars, same name, dashboard only showing 1 on selection to pick from

### Expected Behavior

Have 2 selections on dropdown, even if same name

### Steps To Reproduce

Open dashboard, try to select vehicle, must have 2 with the same name

### Relevant log output

```shell
none
```

### Screenshots

image
image

### Additional data

none

### Type of installation

Docker

### Version

1.28.5

### PR DESCRIPTION
**Pull request by swiffer**
fixes #3891

### CODE REVIEW

**Review by JakobLichterfeld — APPROVED** [review]
Awesome, ty!

**Comment by edgararroyo** [comment]
Nice!  Thank you!

**Comment by arcastro** [comment]
Hi everyone, 

I had previously opened this pull request, for cases of unnamed vehicles
* https://github.com/teslamate-org/teslamate/pull/4198

Looks like the behavior changed in this pull request 

### From:



### To:



In my opinion, displaying a bunch of numbers with no context isn't too great. 

Would you be opposed if I opened a new pull request to change back the case of unnamed vehicles to `VIN $full_vin`?

In summary:
* Uniquely named vehicles: `$Name`
* Duplicate named vehicles: `$Name - $Last_six_of_vin`
* Unnamed vehicles: `VIN $Full_vin`

c.c. @swiffer , @JakobLichterfeld , @edgararroyo

**Comment by swiffer** [comment]
@arcastro - converted into new issue, will be adressed shortly after merge of open PRs
